### PR TITLE
Added an experiment - Oscilloscope

### PR DESCRIPTION
**Pull request by NishithP2004**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **Related Issue**

* **What changes have you introduced?**
Hey there, I have added an experiment for oscilloscope and have redirected the new images to the images folder.

* **Does this PR introduce a breaking change?**

* **Preview / Steps to verify your work**:
https://github.com/NishithP2004/pslab-documentation/blob/master/tutorials/oscilloscope.md

### CODE REVIEW

**Comment by NishithP2004** [comment]
@cweitat @mariobehling  Please review

**Comment by cweitat** [comment]
Don't squash your image.

File: tutorials/oscilloscope.md [hunk 1]
```diff
@@ -52,5 +49,43 @@ In the desktop app you will see something like this:
...
+### Goal of the experiment: To measure the intensity of sound produced.
+
+### Materials required
+* Android Phone
+* [PSLab Android App](https://play.google.com/store/apps/details?id=io.pslab&hl=en_US)
+
+### Procedure
+* Open PSLab Android app 
+
+<img src="/images/img_compass_2.PNG" width="540" height="960">
```
Comment (cweitat) [inline tutorials/oscilloscope.md]: setting width is sufficient

Comment (cweitat) [reply tutorials/oscilloscope.md]: same for other images

**Comment by NishithP2004** [comment]
Done Sir

**Comment by NishithP2004** [comment]
> Please put image files into the /image folder.

I had put it there at ``images``  folder earlier .Could you please recheck.

**Pull request merged**